base/operators.jl

```diff
@@ -827,17 +827,25 @@ julia> [1:5;] |> x->x.^2 |> sum |> inv
 Compose functions: i.e. `(f ∘ g)(args...)` means `f(g(args...))`. The `∘` symbol can be
 entered in the Julia REPL (and most editors, appropriately configured) by typing `\\circ<tab>`.
 
+Function composition also works in prefix form: `∘(f,g)` is the same as `f ∘ g`.
+The prefix form also supports multiple function composition: `∘(f,g,h) = f ∘ g ∘ h`
+and splatting `∘(fs...)` for an iterable.
+
 # Examples
 ```jldoctest
 julia> map(uppercase∘first, ["apple", "banana", "carrot"])
 3-element Array{Char,1}:
  'A'
  'B'
  'C'
+
+julia> fs = (x -> 2x, x -> x/2, x -> x-1, x -> x+1); f = ∘(fs...); f(3) == 3
```

Comment:
I would expand this out so that it's easier to follow the interaction:
```suggestion
julia> fs = [
           x -> 2x
           x -> x/2
           x -> x-1
           x -> x+1
       ];

julia> ∘(fs...)(3)
3.0
```